pyramid/compat.py

```diff
@@ -244,3 +244,20 @@ def unquote_bytes_to_wsgi(bytestring):
 def is_bound_method(ob):
     return inspect.ismethod(ob) and getattr(ob, im_self, None) is not None
 
+def is_unbound_method(fn):
+    """
+    This consistently verifies that the callable is bound to a
+    class.
+    """
+    is_bound = is_bound_method(fn)
+
+    if not is_bound and inspect.isroutine(fn):
+        spec = inspect.getargspec(fn)
+        has_self = len(spec.args) > 0 and spec.args[0] == 'self'
```

Comment:
It's interesting that you brought this up, because when i first read over this code I had the same question, but I let it go because I thought that if someone isn't using `self`, they are writing very unpythonic code...